CefSharp/MouseEvent.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright © 2010-2014 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace CefSharp
+{
+    public class MouseEvent : IMouseEvent
```

Comment:
Do we need the `IMouseEvent` interface? Can probably make this a struct as well (There's a few other classes in the code base that probably should be structs, one day I'll get around to changing them)